src/custom/state/appData/updater.ts

```diff
@@ -16,8 +16,9 @@ import {
 } from 'state/appData/types'
 
 const UPLOAD_CHECK_INTERVAL = ms`10s`
-// TODO: implement exponential back off instead of fixed retry time
-const TRY_AGAIN_AFTER = ms`1min`
+const BASE_TIME_BETWEEN_ATTEMPTS = 2 // in seconds, converted to milliseconds later
```

Comment:
```suggestion
const SECONDS_BETWEEN_ATTEMPS = 2 // in seconds, converted to milliseconds later
```

Reply:
`BASE_FOR_EXPONENTIAL_BACKOFF` might describe it better, will use that

Just a `2` I would say to myself: "magic number, wtf?"